spec/models/location_spec.rb

```diff
@@ -1016,15 +1016,16 @@
       end
 
       it "returns reactivating soon if the location has a future reactivation date" do
-        FactoryBot.create(:location_deactivation_period, deactivation_date: Time.zone.yesterday, reactivation_date: Time.zone.tomorrow, location:)
+        deactivation_period = FactoryBot.create(:location_deactivation_period, deactivation_date: Time.zone.yesterday, reactivation_date: Time.zone.tomorrow, location:)
         location.save!
+        expect(deactivation_period.deactivation_date).to eq(Time.zone.yesterday.to_date)
```

Comment:
The tests for this are flaky and instead of `location.status` being `reactivating_soon` it ends up being `active`. I initially thought it's a time issue, but couldn't trigger a failure when setting different times. I'm not sure why this is happening, there isn't an obvious way to reproduce it on the review apps, so adding this line just to check if the deactivation period does get created, but the status is still `active`